adjust mobility model

sven0311 opened the pull request:
nodes are staying for 5 min in the same state
nodes are not doing rwp in the most areas
nodes are moving to the ubahn after exit state and are deactivated then

Commit:
nodes are moving to the ubahn after exit state and are deactivated then
nodes are staying for 5 min in the same state
nodes are not doing rwp in the most areas

Commit:
fix performance issue

geitner-max reviewed (changes requested):
Beim Durchlaufen habe ich folgende Nullpointerexception erhalten:
Exception in thread "main" java.lang.NullPointerException
	at movement.StatefulRwp.updateState(StatefulRwp.java:225)
	at movement.StatefulRwp.getPath(StatefulRwp.java:47)
	at core.DTNHost.setNextWaypoint(DTNHost.java:418)
	at core.DTNHost.move(DTNHost.java:396)
	at core.World.moveHosts(World.java:214)
	at core.World.update(World.java:164)
	at gui.DTNSimGUI.runSim(DTNSimGUI.java:113)
	at ui.DTNSimUI.start(DTNSimUI.java:77)
	at core.DTNSim.main(DTNSim.java:92)

Da fehlt irgendwo noch eine state==null Abfrage.

sven0311 commented:
@geitner-max du hast glaub ich nicht den neusten Stand. Das sollte behoben sein

geitner-max commented:
Stimmt, da gab es noch einen neuen Commit. Jetzt geht alles. Passt!

geitner-max committed:
enhanced InfectionReport class and modified group parameters
- reduced probability for infection
- added distance factor between hosts that influences viral load for each message
- infected people can't get infected again
- increased to 100 simulated hosts
- added labels to InfectionReport output for each infected person

geitner-max committed:
hosts in an inactive movement state can't get infected
